uint8 LEFT_ARM_ONLY=1
uint8 RIGHT_ARM_ONLY=2
uint8 BOTH_ARMS=3
uint8 ArmOption
uint8 COST_ONLY=1
uint8 BRITTLENESS_ONLY=2
uint8 COST_AND_BRITTLENESS=3
uint8 FeatureOption
float64[] LeftArmConfiguration
float64[] RightArmConfiguration
float64 GradientMultiplier
bool UpdateSDF
---
uint8 SUCCESS=0
uint8 INVALID_REQUEST=1
uint8 Status
float64 LeftArmCost
float64 RightArmCost
float64 LeftArmBrittleness
float64 RightArmBrittleness
